nevergrad/optimization/optimizerlib.py

```diff
@@ -7,6 +7,7 @@
 from collections import deque
 import warnings
 import cma
+from fcmaes import cmaes
```

Comment:
This adds a dependency but has not been added to the requirements.
I'd try to avoid adding dependencies when I can, so, I would recommend making this step by step:
for now we would add fcmaes as a dependency in requirements/bench.txt so that it is always available at least in the benchmarks.

Then in order to avoiding breaking all the code when this is not installed,  add the import inline:
```
@property
def es(self) -> tp.Any:  # typing not possible since not imported :(
try:
    from fcmaes import cmaes
except ImportError as e:
    raise ImportError("Please install fcmaes (pip install fcmaes) to use FCMA optimizers") from e
    if self._es is None:
        ...
```

That would make it a soft dependency for now, and eventually if it becomes more and more relevant compared to the current CMA, we could move it to a hard dependency.

Reply:
Sounds reasonable. Will do as you propose.

Reply:
Sounds reasonable. Will do as you propose.

Reply:
ci/circleci check still fails since the soft dependency is not fulfilled. 
Requires the change in requirements/bench.txt. Shall I add this dependency as 
part of the PR?

Reply:
Yes please do. bench.txt requirements are "extra" requirements, installed if required by the users.